### ISSUE TITLE
Scalability test for AWS

### ISSUE THREAD

**Issue by mcwhitaker** [avalonmediasystem/avalon#1247]
## Done looks like
Scalability reports generated, and based on those results, create a separate task if necessary.

## Tasks
- [ ] determine which load testing tool we'll use (IU's Neoload or a third party)
- [ ] load test data into the AWS Avalon so that we have parts with sufficient complexity
- [ ] run testing script

**Comment by carrickr** [comment]
Note this story is tasked, by blocked by all the other AWS items and the need to have a functioning AWS install

**Comment by davidschober** [comment]
@mcwhitaker @joncameron  We dug into load testing AWS in our current infrastructure and it's a bit problematic. Cloud front (being a very efficient edge cache) takes most of the work away from the core infrastructure when you load test in normal ways (a single machine). 

REFERENCE: 
http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/load-testing.html

We would need to create a geographically disparate set of machines to load test effectively. This could be done using something like bees with machine guns <https://github.com/newsapps/beeswithmachineguns>. 

That would mean incurring a cost for the Micro Instances. NU or IU would have to discuss. 

We may want to remove points and abandon .

**Comment by mcwhitaker** [comment]
@davidschober Understood. Thanks for the document reference. Would it still be worth to just throw a load and confirm that it all works as expected, i.e., that cloudfront takes care of it? I am just asking, I am not advocating.

**Comment by davidschober** [comment]
I think I'm going to abandon.
